mwdb/resources/file.py

```diff
@@ -470,6 +470,124 @@ def post(self, identifier):
         return schema.dump({"token": download_token})
 
 
+@rate_limited_resource
+class FileDownloadResourceXored(Resource):
+    """copy of FileDownloadResource with file_obj.iterate_xored() in .get method"""
+    def get(self, identifier):
```

Comment:
This is almost vebatim copy of the FileDownloadResource (even docstrings are not changed as far as I can see). The only thing changed is the iterate_xored in the return statement. Is it possible to reduce the duplication somehow?

A maintainer (@psrok1 ?) would have to step down to decide what's the best solution, but maybe just ditch this endpoint altogether, and instead pass an optional parameter `?obfuscate=true` to the regular download endpoint? Implementing this should be way less bloated on the server side, and equally easy to use on the client side.

Reply:
General idea is OK, I know it's common problem for some users. That's why we also introduced encrypted zip download. As you already noticed: AV solutions are strongly allergic to artifacts containing malicious code that are stored on disk.

I also think that parametrizing that xor with `obfuscate=1` is actually a nice idea instead of introduction of another endpoint.